server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -18,9 +18,18 @@
 import com.beust.jcommander.JCommander;
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.ParameterException;
-import io.seata.common.util.StringUtils;
 import io.seata.config.ConfigurationFactory;
 import io.seata.core.constants.ConfigurationKeys;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
```

Comment:
recommended to extend the StringUtils and NumberUtils of seata

Reply:
There is not `NumberUtils` exist in project, and `StringUtils` doesn't have `defaultIfBlank` method.
So used apache utility library.

Reply:
Relying on commons, easy jar version conflict